.github/workflows/delete-v2-review-app.yml

```diff
@@ -0,0 +1,110 @@
+name: Delete v2 Review App
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+jobs:
+  delete-v2-review-app:
+    name: Delete v2 Review App ${{ github.event.pull_request.number }}
+    concurrency: deploy_v2_review_${{ github.event.pull_request.number }}
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'deploy_v2') || contains(github.event.pull_request.labels.*.name, 'dependencies') }}
+    runs-on: ubuntu-latest
+    environment: review_aks
+    steps:
+      - name: Exit whole workflow if wait was not successful
+        if: ${{ steps.wait_for_deployment.outputs.conclusion != '' && steps.wait_for_deployment.outputs.conclusion != 'success' }}
+        run: exit 1
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Use Terraform v1.2.3
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.2.3
```

Comment:
Future improvement: might be worth getting this dynamically as implemented [here](https://github.com/DFE-Digital/refer-serious-misconduct/blob/f12220ee2fb8c846e47498c964a582ede0103e4d/.github/actions/deploy-environment/action.yml#L36) as this value is hardcoded in at least 4 places

Reply:
https://trello.com/c/aExRfBEy